hack/terraform-quickstart/main.tf

```diff
@@ -16,17 +16,18 @@ resource "aws_instance" "bootstrap_node" {
   associate_public_ip_address = true
   depends_on                  = ["aws_internet_gateway.main"]
 
-  tags {
-    Name = "${var.resource_owner}"
-  }
+  tags = "${map(
+    "Name", "${var.resource_owner}",
+    "kubernetes.io/cluster/${var.resource_owner}", "owned"
+  )}"
 
   root_block_device {
     volume_type = "gp2"
     volume_size = "30"
   }
 
   provisioner "file" {
-    source = "environment_${var.environment}.txt"
+    source      = "environment_${var.environment}.txt"
```

Comment:
Ah, we should make sure we `terraform fmt` all of these files as part of CI.

cc @rphillips